xml_inscriptions/transcribed/NY.NY.MMA.LG.74.51.2393.xml

```diff
@@ -0,0 +1,195 @@
+                            </p>
+                        </provenance>
+                        
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Myres"/>
+                        <biblScope unit="item">no. 1927</biblScope>
+                    </bibl>
+                    
+                    <bibl> 
+                        <ref type="eternal" target="https://www.metmuseum.org/art/collection/search/241947">Metropolitan Museum Web Entry</ref> 
+                    </bibl>
+                    
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+        
+        <encodingDesc>
+            
+            <xi:include
+                href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for US Epigraphy controlled values</p>
+                </xi:fallback>
+            </xi:include>
+            
+        </encodingDesc>
+        
+        <profileDesc> <!-- not used --> </profileDesc>
+        
+        
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+        <!-- Record all changes here -->
+        
+        <revisionDesc>
+            <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+            <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+            <change when="2020-11-24" who="Itamar Levin">Transcription, translation, and
+                metadata</change>
+        </revisionDesc>
+        
+    </teiHeader>
+    
+    <!--  *************************************  <facsimile>  *********************************  -->
+    <!-- Used to record images of an inscription.   -->
+    
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.LG.74.51.2393.jpg"/>
+        </surface>
+    </facsimile>
+    
+    <text>
+        <body>
+            <div type="edition" subtype="section" n="i" xml:lang="ltn">
+                <ab>
+                    <lb n="1"/><persName><name key="Iulia" type="praenomen">Iulia</name> <name type="praenomen" key="Olympus">Olym
+                        <lb n="2" break="no"/>pi</name> <expan><abbr>l</abbr><ex>ibertata</ex></expan> <name key="Donata" type="cognomen">Donata</name></persName> 
+                        <lb n="3"/><expan><abbr>h</abbr><ex>ic</ex></expan> <expan><abbr>p</abbr><ex>osita</ex></expan> est.  
+                </ab></div>
+            
+            <div type="edition" subtype="section" n="ii" xml:lang="grk">
```

Comment:
xml:lang code is grc, as above; also only use separate sections if you are considering it to be two separate texts. My instinct is also to maintain the line numeration here, rather than resetting.